config/consensus.go

```diff
@@ -1146,11 +1155,15 @@ func initConsensusProtocols() {
 
 	vFuture.LogicSigVersion = 7
 	vFuture.MinInnerApplVersion = 4
-
 	vFuture.UnifyInnerTxIDs = true
 
 	vFuture.EnableSHA256TxnCommitmentHeader = true
 
+	// Boxes (unlimited global storage)
+	vFuture.BoxFlatMinBalance = 2500
```

Comment:
This seems high per byte relative to 2k bytes you get for 0.1 when using lsig account storage

Reply:
I thought I replied but don't see it, so forgive me if you see two replies.

I think it "costs" a full algo to get 2k with a logicsig. 0.1 to fund it. 0.1 base to opt in to the app that wants more storage. Then there's the 0.050 algos per byte string in the local schema. So 16 (max allowed, given 2k bytes) is another 0.8A).

Reply:
ah of course, so 1A/2kb with acct hackery vs 5A/2kb for boxes

Reply:
A 2kb box ought to only be about 0.8A, not 5A.  2500+2048*400 = 821,700.

Reply:
🤦 i'd been reading this wrong thinking 2500/byte this whole time, ignore entire thread